models/reduce_pretrained_ft.ipynb

```diff
@@ -0,0 +1,401 @@
+{
```

Comment:
<p>Why do we need to retain 70% of the total singular value sizes? Could you please add an explanation here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/github-labeler/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>